src/frontend/web/src/ratingpage/EffortBar.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import './EffortBar.css';
+
+const EffortBar = (props) => {
+
+    const max_effort = 5;
+    const full_elements = [];
+    const empty_elements = [];
+
+    for(let i = 0; i < max_effort; i = i + 1){
+      if(i < Number(props.effort)){
```

Comment:
Here you could change it to be written without using `props`
```suggestion
      if(i < Number(effort)){
```